### ISSUE TITLE
Suggestion: Container sorting and stack consolidation

### ISSUE THREAD

**Issue by LiaungYip** [StrangeLoopGames/EcoIssues#4307]
Currently, it's possible to get containers into a state like this, with no ordering of contents and multiple partial stacks of the same item.

![image](https://user-images.githubusercontent.com/2173684/36072189-17f41484-0f67-11e8-982e-e11fe8f6e23c.png)

It would save player time (and sanity) if there were a "sort all container contents" button, which would sort all items alphabetically by name, and combine partial stacks into full stacks. Like this:

![image](https://user-images.githubusercontent.com/2173684/36072214-5c7a9736-0f67-11e8-9c8a-3be21c06e516.png)

This should operate on all linked storages visible in the Storage pane.

This doesn't affect game balance as no items are ever moved between storage containers - only re-arranged within the storage container they're already in. It's purely UI sugar to make the play experience a bit nicer.

**Comment by MarjieAVolk** [comment]
Yeah I almost think inventories should do this automatically.  Maybe not your backpack, but stockpiles.
